src/action/shutdown.rs

```diff
@@ -130,6 +134,7 @@ pub struct Shutdown {
     pub(crate) immediate: bool,
 }
 
+#[export_doc(shutdown)]
```

Comment:
Another corner case: we want to generate documentation for the options for `shutdown`, but there's no corresponding options object.